### ISSUE TITLE
Update Phoenix dependency to test 1.3.0-rc

### ISSUE THREAD

**Issue by cbarratt** [appsignal/appsignal-elixir#163]
Hi guys,

I guess with it being RC and not being fully released I can totally see why AppSignal is locked to 1.2.0, though I guess it may be worth while having a release/branch that has a more relaxed version so this could be tested before full Phoenix 1.3.0 release?

```
Failed to use "phoenix" (version 1.3.0-rc.0) because
  apps/maggie/mix.exs requires ~> 1.3.0-rc
  appsignal (version 1.0.0) requires ~> 1.2.0
```

**Comment by jeffkreeftmeijer** [comment]
Hey Callum,

You're right, we locked the version to ~> 1.2.0 to have some time to test with the RC before officially supporting it. After reading your comment, we decided to open this up a bit. Here's what we're doing:

- We're changing the Phoenix dependency [from `~> 1.2.0` to `>= 1.2.0`](https://github.com/appsignal/appsignal-elixir/pull/167), so we'll support 1.3.0 when it comes out. This still excludes prereleases.
- We've added a [`phoenix-pre` branch](https://github.com/appsignal/appsignal-elixir/tree/phoenix-pre), based on master, which we'll use to test support for upcoming Phoenix releases. It depends on Phoenix `>= 1.3.0-rc.0`.

To try Appsignal on a Phoenix prerelease, you can use the `phoenix-pre` branch in your project:

``` elixir
[{:appsignal, git: "git@github.com:appsignal/appsignal-elixir.git", branch: "phoenix-pre"}]
```

**Comment by casio** [comment]
fyi just stumbled upon a `Host key verification` problem using `git@github.com` in a travis build setup.
`{:appsignal, github: "appsignal/appsignal-elixir", branch: "phoenix-pre"}` as dep is more usable, I guess

**Comment by jeffkreeftmeijer** [comment]
Hey @casio, Thanks for that, I've updated my original comment. :)

**Comment by jeffkreeftmeijer** [comment]
Since Phoenix 1.3 has now been released, AppSignal for Elixir now automatically supports it. Instead of using the `phoenix-pre` branch, you can now switch to the released version by using `{:appsignal, "~> 1.0"}` as your dependency.
